frontend/sw360-portlet/src/main/java/org/eclipse/sw360/portal/portlets/projects/ProjectPortlet.java

```diff
@@ -1793,13 +1797,16 @@ private boolean addToVulnerabilityRatings(Map<String, Map<String, VulnerabilityR
 
             VulnerabilityCheckStatus vulnerabilityCheckStatus = vulnerabilityCheckStatusHistory.get(vulnerabilityCheckStatusHistory.size() - 1);
             VulnerabilityRatingForProject rating = vulnerabilityCheckStatus.getVulnerabilityRating();
+            String action = vulnerabilityCheckStatus.getAction();
```

Comment:
Can vulnerabilityCheckStatus be null? if so please consider the null check.

Reply:
It can not be `null` considering prior checks